tests/testthat/test-stanmodule.R

```diff
@@ -39,20 +38,17 @@ test_that("Loading from file works as expected", {
 
 
 test_that("StanModules can be rendered as a string", {
-
     x <- StanModule(
         data = c("some data"),
         functions = c("some code", "some more code")
     )
 
     expected <- paste0(
         c(
-            "",
             "functions {",
             "some code",
             "some more code",
             "}",
-            "",
```

Comment:
@G-Kazantzidis ,

Did you mean to delete this one, based on your last comment I would have expected there to be a blank line here because its between 2 blocks ?

Reply:
this is result of the `h_bracket` function 

https://github.com/Genentech/jmpost/blob/f8a2e4eaa49116fdd79530f519687da8a7605458/R/h_bracket.R#L13

Reply:
Sorry I'm not sure I follow surely we want a space between blocks i.e.

```
data {
code
}

functions {
code
}
```

As opposed to:

```
data {
code
}
functions {
code
}
```

Reply:
I assume that this makes the stan code slightly more readable. Thus, I will go for the first option with the additional lines.

Reply:
yer apologies I appreciate this is pretty petty stuff now but is good to get it as nice as possible :)